lib/global-config.js

```diff
@@ -125,6 +125,11 @@ function save() {
 exports.save = save;
 
 exports.set = function(name, val) {
+ 
+  if(!(name && val)) {
+     console.warn('Please pass option and setting value! See for more information jspm --help');
+  }
```

Comment:
This check should be in https://github.com/jspm/jspm-cli/blob/0.17/cli.js#L590 I think rather, where we could perhaps include something like:

```
ui.warn('jspm config requires a property and value via %jspm config property value%')
```

or similar.